database_cleanup/models/purge_columns.py

```diff
@@ -20,7 +20,12 @@ def purge(self):
         """
         Unlink columns upon manual confirmation.
         """
-        for line in self:
+        if self:
+            objs = self
+        else:
```

Comment:
PS: If you really need this for whatever reason, just assign to `self` and leave the rest of the code as it is. It's just a variable like any other